shell/platform/darwin/ios/framework/Source/FlutterPlatformPlugin.mm

```diff
@@ -76,8 +76,22 @@ - (void)playSystemSound:(NSString*)soundType {
   }
 }
 
-- (void)vibrateHapticFeedback {
-  AudioServicesPlaySystemSound(kSystemSoundID_Vibrate);
+- (void)vibrateHapticFeedback:(NSString*)feedbackType {
+  if (!feedbackType) {
```

Comment:
`if (!feedbackType || !@available(iOS 10, *))`

Reply:
ya, I did that initially but on second thought, I think we wouldn't want to buzz the vibrator on every picker scroll and similar circumstances. On iOS, the haptic engine type feedbacks like on pull to refresh etc are just ignored on older devices.